omod/src/main/webapp/fragments/patientdashboard/contactInfoInline.gsp

```diff
@@ -4,12 +4,12 @@
 
 <div class="contact-info-inline">
     <span>
-        ${ ui.format(config.patient.personAddress).replace("\n", ", ")}
+        ${ ui.encodeHtmlContent(ui.format(config.patient.personAddress).replace("\n", ", "))}
         <em>${ ui.message("coreapps.person.address")}</em>
     </span>
     <span class="left-margin">
         <span id="coreapps-telephoneNumber">
-            ${config.patient.telephoneNumber ?: ''}
+            ${ ui.encodeHtmlContent(config.patient.telephoneNumber ?: '') }
```

Comment:
I was able to insert js into the phone number field by crafting a malicious POST request to an existing patient's editSection page. I used Tamper Data (a free Firefox addon) to edit the POST request. The attached image was taken as I was editing the POST request just after hitting 'confirm' on the 'edit contact info' page.

![tele-xss](https://cloud.githubusercontent.com/assets/7016326/20863694/06b6dbc4-b9a1-11e6-8af6-68e2132cf2b6.png)